clang/include/clang/AST/Stmt.h

```diff
@@ -460,10 +460,10 @@ class alignas(void *) Stmt {
     unsigned : NumExprBits;
 
     static_assert(
-        llvm::APFloat::S_MaxSemantics < 16,
-        "Too many Semantics enum values to fit in bitfield of size 4");
+        llvm::APFloat::S_MaxSemantics < 32,
+        "Too many Semantics enum values to fit in bitfield of size 5");
     LLVM_PREFERRED_TYPE(llvm::APFloat::Semantics)
-    unsigned Semantics : 4; // Provides semantics for APFloat construction
```

Comment:
Didn't I see this in another PR you sent? If not, is there a way to separate this into PRs for LLVM/Clang and PRs for MLIR?

Reply:
This code is from [PR-97179](https://github.com/llvm/llvm-project/pull/97179). Need to use one additional byte to store new item with id 16 (total 17 items)